doc/langdef.md

```diff
@@ -712,26 +740,33 @@ All predefined operators, functions and constants are listed in the table below.
 For each symbol, the available overloads are listed. Operator symbols use a
 notation like `_+_` where `_` is a placeholder for an argument.
 
-TODO https://issuetracker.google.com/67014381 : have better descriptions. The
-table is auto-generated so the descriptions need to be updated in the code.
-
 ### Timezones
 
-Timezones are expressed in the following grammar: `grammar TimeZone = "UTC" |
-LongTZ | FixedTZ ; LongTZ = ? list available at
-http://joda-time.sourceforge.net/timezones.html ? ; FixedTZ = ( "+" | "-" )
-Digit Digit ":" Digit Digit ; Digit = "0" | "1" | ... | "9" ;`
+Timezones are expressed in the following grammar:
+
+```grammar
+TimeZone = "UTC" | LongTZ | FixedTZ ;
+LongTZ = ? list available at
+           http://joda-time.sourceforge.net/timezones.html ? ;
+FixedTZ = ( "+" | "-" ) Digit Digit ":" Digit Digit ;
+Digit = "0" | "1" | ... | "9" ;
+```
 
 Fixed timezones are explicit hour and minute offsets from UTC. Long timezone
 names are like `Europe/Paris`, `CET`, or `US/Central`.
 
 ### Regular Expressions
 
 Regular expressions follow the
-[RE2 syntax](https://github.com/google/re2/wiki/Syntax).
+[RE2 syntax](https://github.com/google/re2/wiki/Syntax). Regular expression
+matches succeed if they match a substring of the argument. Use explicit anchors
+(`^` and `$`) in the pattern to force full-string matching, if desired.
 
 ### List of Standard Definitions
 
+TODO https://issuetracker.google.com/67014381 : have better descriptions. The
```

Comment:
This issue has been archived, but there is one currently open within the cel-go repository to create a better docgen tool.

Reply:
Updated the TODO.